easybuild/easyconfigs/d/DualSPHysics/DualSPHysics-5.0.175.patch

```diff
@@ -0,0 +1,25 @@
+diff -Naur DualSPHysics-5.0.175/src/source/CMakeLists.txt DualSPHysics-5.0.175/src/source/CMakeLists.txt
```

Comment:
Please add the author + short description for this patch at the top (and then update its checksum in the easyconfigs).